assets/schema_database.json

```diff
@@ -28,7 +27,6 @@
             },
             "db_name": {
```

Comment:
I wonder if this should also be made unique per tool too

Reply:
I think `"uniqueEntries": ["tool", "db_name"]` is sufficient. Different databases can be used with the same tool. This could be useful for certain purposes, for instance database validations.